### PR TITLE
Fix dropout train_phase argument propagation

### PR DESCRIPTION
**Pull request by Kismuz**
Cheers, thank you for great job!
Here is minor fix enabling proper `training` indicator propagation to SpatialDropout layer in case when TCN layer is used outside keras.Model optimization loop and train phase can not be inferred from Keras backend.

### CODE REVIEW

**Comment by philipperemy** [comment]
@Kismuz thanks for your contribution :)
LGTM!

**Comment by philipperemy** [comment]
Pushed to 2.8.3 https://github.com/philipperemy/keras-tcn/commit/154cda58d0ee4d637805c0edb67b6078b697a853
